/diffdrive_kin_trajctrl:

    # run period of the node
    run_period: 0.01

    # distance of point P from odometric center
    P_dist: 0.1

    # PID parameters for trajectory tracking
    Kp: 0.8               # proportional gain
    Ki: 0.8               # integral gain
    Kd: 0.0               # derivative gain

